bin/doc8

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import sys
+
+__requires__ = ['doc8', 'sphinx']
+import pkg_resources
+
+import doc8.main
+from sphinx import directives
+
+# These setup functions call docutils.parsers.rst.directives.register_directive
+# which is then used by the docutil during validation
+# by the restructuredtext_lint package.
+setup_funcs = [
+    directives.setup,
+    directives.code.setup,
+    # directives.other.setup,
```

Comment:
That one is commented out because it's problematic. (I think I mention why this exists in the extended commit message for this change.) This cli script is wrapped to include these directives, where they were not previously included, which causes validation errors.

Reply:
ok